tests/xtro-sharpie/macOS-AppKit.todo

```diff
@@ -4,12 +4,3 @@
 !missing-selector! +NSColor::findHighlightColor not bound
 !missing-selector! +NSColor::placeholderTextColor not bound
 !missing-selector! NSSliderCell::setImage: not bound
-## appended from unclassified file
```

Comment:
what about the other todo (they are from past beta) ?

Reply:
Unsure, they were not in this b4->b5 and I didn't see in b4 either. I'll look at them outside of this PR.